### ISSUE TITLE
How should I set "markSeen: true" option correctly?

### ISSUE THREAD

**Issue by anotheri** [circuithub/mail-listener#13]
#### Problem

I've got the first e-mail and `mail-listener` fetch it. But mail after fetching stay `UNSEEN`.
I've got the second e-mail and `mail-listener` fetch all `UNSEEN` e-mails in the "INBOX" (by default) mailbox (first and second). And two e-mails (first and second) stay `UNSEEN`. And so on.
#### What I need

Incoming e-mails should marks as `SEEN` on fetch.
#### Search for solutions

In the `imap` `fetch` funcion there is a `markSeen` option. But it's not work correct, when I just add `markSeen: true` here:

```
#5. fetch emails
        @imap.fetch searchResults, 
          headers:
            parse: false
          body: true
          markSeen: true // ← added string!
          cb: (fetch) => 
            ...
```

It have only next output in console:

```
16 Apr 10:35:54 - new mail arrived with id 1
16 Apr 10:35:54 - found 1 emails
```

And incoming e-mail still stay `UNSEEN`.
#### Question

Could you help me with this problem? How should I set `markSeen: true` option in the `fetch` command options for it correct working? And I think it will be good if there is a posibility to set this option via `mail-listener` options. 

Thank you.

**Comment by anotheri** [comment]
#### This is correct solution

```
# 5. fetch emails
        @imap.fetch searchResults, {
            markSeen: true
          },
          headers:
            parse: false
          body: true
          cb: (fetch) =>
            ...
```

So, please, add a posibility to set this option via `mail-listener` options.

**Comment by chirag04** [comment]
+1

**Comment by ankitsny** [comment]
`
    imap.openBox("INBOX", false, function(err, mailBox) {
             if (err) {
            console.error(err);
            return;
        }
        imap.search(["UNSEEN"], function(err, results) {

            imap.setFlags(results, ['\\Seen'], function(err) {
                if (!err) {
                    console.log("marked as read");
                } else {
                    console.log(JSON.stringify(err, null, 2));
                }
            });

            var f = imap.seq.fetch(results, { bodies: "" });
            f.on("message", processMessage);
            f.once("error", function(err) {
                return Promise.reject(err);
            });
            f.once("end", function() {
                console.log("Done fetching all unseen messages.");
                imap.end();
            });
        });
    });`

processMessage

`function processMessage(msg seqno){
         //use mailparser
}`


I don't know why my code is not getting properly formatted here,
so basically results contains the uuid

we need to call imap.setFlags([uuids], ['\\Seen'], cb) function to make it mark as read
